modules/osdk-updating-projects.adoc

```diff
@@ -0,0 +1,353 @@
+	$(call go-get-tool,$(ENVTEST),sigs.k8s.io/controller-runtime/tools/setup-envtest@latest)
+
+# go-get-tool will 'go get' any package $2 and install it to $1.
+PROJECT_DIR := $(shell dirname $(abspath $(lastword $(MAKEFILE_LIST))))
+define go-get-tool
+@[ -f $(1) ] || { \
+set -e ;\
+TMP_DIR=$$(mktemp -d) ;\
+cd $$TMP_DIR ;\
+go mod init tmp ;\
+echo "Downloading $(2)" ;\
+GOBIN=$(PROJECT_DIR)/bin go get $(2) ;\
+rm -rf $$TMP_DIR ;\
+}
+endef
+---- 
++
+.New `Makefile`
+[source,terminal]
+----
+##@ Build Dependencies
+
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
+## Tool Binaries
+KUSTOMIZE ?= $(LOCALBIN)/kustomize
+CONTROLLER_GEN ?= $(LOCALBIN)/controller-gen
+ENVTEST ?= $(LOCALBIN)/setup-envtest
+
+## Tool Versions
+KUSTOMIZE_VERSION ?= v3.8.7
+CONTROLLER_TOOLS_VERSION ?= v0.8.0
+
+KUSTOMIZE_INSTALL_SCRIPT ?= "https://raw.githubusercontent.com/kubernetes-sigs/kustomize/master/hack/install_kustomize.sh"
+.PHONY: kustomize
+kustomize: $(KUSTOMIZE) ## Download kustomize locally if necessary.
+$(KUSTOMIZE): $(LOCALBIN)
+	curl -s $(KUSTOMIZE_INSTALL_SCRIPT) | bash -s -- $(subst v,,$(KUSTOMIZE_VERSION)) $(LOCALBIN)
+
+.PHONY: controller-gen
+controller-gen: $(CONTROLLER_GEN) ## Download controller-gen locally if necessary.
+$(CONTROLLER_GEN): $(LOCALBIN)
+	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-tools/cmd/controller-gen@$(CONTROLLER_TOOLS_VERSION)
+
+.PHONY: envtest
+envtest: $(ENVTEST) ## Download envtest-setup locally if necessary.
+$(ENVTEST): $(LOCALBIN)
+	GOBIN=$(LOCALBIN) go install sigs.k8s.io/controller-runtime/tools/setup-envtest@latest
+---- 
+endif::[]
+
+ifdef::golang,hybrid[]
+.. Update `ENVTEST_K8S_VERSION` and `controller-gen` fields in your `Makefile` to support Kubernetes 1.24:
++
+[source,terminal]
+----
+...
+ENVTEST_K8S_VERSION = 1.24 <1>
+...
+sigs.k8s.io/controller-tools/cmd/controller-gen@v0.9.0 <2>
+----
+<1> Update version `1.22` to `1.24`.
+<2> Update version `0.7.0` to `0.8.0`.
+endif::[]
+
+.. Apply the changes to your `Makefile` and rebuild your Operator by entering the following command:
++
+[source,terminal]
+----
+$ make
+----
+
+ifdef::golang,hybrid[]
+. Make the following changes to the `go.mod` file to update Go and its dependencies:
++
+[source,golang]
+----
+go 1.18 <1>
+
+require (
+  github.com/onsi/ginkgo v1.16.5 <2>
+  github.com/onsi/gomega v1.18.1 <3>
+  k8s.io/api v0.24.0 <4>
+  k8s.io/apimachinery v0.24.0 <4>
+  k8s.io/client-go v0.24.0 <4>
+  sigs.k8s.io/controller-runtime v0.12.1 <5>
+)
+----
+<1> Update version `1.16` to `1.18`.
+<2> Update version `v1.16.4` to `v1.16.5`.
+<3> Update version `v1.15.0` to `v1.18.1`.
+<4> Update version `v0.22.1` to `v0.24.0`.
+<5> Update version `v0.10.0` to `v0.12.1`.
+endif::golang,hybrid[]
+
+ifdef::hybrid[]
+. Update the Helm Operator plug-ins:
++
+[source,golang]
+----
+github.com/operator-framework/helm-operator-plugins v0.0.12-0.20220608155702-d3967d2ae2a <1>
```

Comment:
Should 
update 
"github.com/operator-framework/helm-operator-plugins v0.0.8"
to 
"github.com/operator-framework/helm-operator-plugins v0.11.0"

Reply:
Addressed. Would you please double check this one?